python/foxglove-sdk/src/websocket.rs

```diff
@@ -24,6 +24,23 @@ pub struct PyChannelView {
     topic: Py<PyString>,
 }
 
+/// Information about a client channel.
+#[pyclass(name = "ClientChannel", module = "foxglove")]
+pub struct PyClientChannel {
+    #[pyo3(get)]
```

Comment:
You could add a [`get_all` parameter](https://pyo3.rs/v0.23.5/class.html?highlight=get_all#customizing-the-class) to the `#[pyclass]` attribute above if you want to avoid all of these individual getters. (We could do it elsewhere too; this just has a lot of them.)